faasmcli/faasmcli/tasks/codegen.py

```diff
@@ -52,6 +52,7 @@ def local(ctx, wamr=False):
 
     # Always run the codegen required by the tests
     codegen(ctx, "demo", "echo", wamr=True)
+    codegen(ctx, "demo", "chain", wamr=True)
```

Comment:
Eventually we'll run all the tests with both WAMR and WAVM, but for now we can just be tactical with WAMR